Cargo.toml

```diff
@@ -80,6 +83,9 @@ git = "https://github.com/pingcap/rust-prometheus.git"
 default-features = false
 features = ["nightly", "push"]
 
+[dependencies.grpc]
+git = "https://github.com/pingcap/grpc-rust.git"
```

Comment:
Latest upstream uses crate `bytes` which is newer then TiKV's, and it seems that they are incompatible, I will resolve it later.